packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.tsx

```diff
@@ -204,6 +213,7 @@ export class GuxTooltipBase {
         id={this.tooltipId ? undefined : this.id}
         role={this.tooltipId && !this.isShown ? undefined : 'tooltip'}
```

Comment:
Is it bad to have aria-hidden with role tooltip? I don't have any strong thoughts on it but just wondering

Reply:
It didn't seem to affect anything in my testing with voiceover or in the scan with axe, so I guess it's fine?